PHAREHUB/PHARE
# MPI fixes/test disablements/etc

```
- CMake add_phare_test to allow runtime test execution method override i.e. with mpirun

- fix particle data tests except refine test for now

-  refine particle test cannot be ran in parallel as designed
-  Level 1 cannot have contiguous patches

-  test interior works with 2 cores as expected

-  following reviews on PR https://github.com/PHAREHUB/PHARE/pull/56

-  disable tests and fix bugs related to mpi usge

-  disable field streaming test, see github issue #60
```
